11/umbraco-cms/reference/configuration/contentsettings.md

```diff
@@ -170,6 +171,10 @@ This setting is used for controlling whether or not the Data Types marked as obs
 
 By default this is set to `false`. To make the obsolete data types visible in the dropdown change the value to `true`.
 
+### Show Domain Warnings
+
+If you have a multilingual Umbraco site and you haven't configured Domains correctly for each language then every time you publish you will get the following warning: "Content published: Domains are not configured for multilingual site, please contact an administrator, see log for more information" - if however, you have a legitimate use case for not setting a Domain for a certain circumstance, these warnings every time an editor publishes a page can be annoying. This setting ShowDomainWarnings can be set to 'false' to stop them from displaying.
```

Comment:
When writing documentation we have a style guide to follow so the documentation is more readable. One of the guides is to make the [sentences shorter than 25 words](https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences), therefore I am suggesting the following: 
```suggestion
If you do not configure Domains for each language in a multilingual site then every time you publish your content you get this warning:

`Content published: Domains are not configured for multilingual site, please contact an administrator, see log for more information.`

If you have a use case for not setting the domains, you can set this setting **ShowDomainWarnings** to `false` to stop the warning from displaying.
```